L/libcoreblas/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcoreblas"
+version = v"23.8.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Rabab53/CoreBlas.git", "a03f4a518e8896cfbc0712cb71f0420449daf592")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+BLAS_NAME=blastrampoline
+if [[ "${target}" == *-mingw* ]]; then
+    BLAS_LIB=${BLAS_NAME}-5;
+else
+    BLAS_LIB=${BLAS_NAME};
+fi
+
+if [[ ${nbits} == 64 ]]; then
+    CMAKE_OPTIONS=(-DCOREBLAS_USE_64BIT_BLAS=ON);
+else
+CMAKE_OPTIONS=(-DCOREBLAS_USE_64BIT_BLAS=OFF);
+fi
+
+cd CoreBlas/
+
+ cmake -B build -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release \
+        -DBLAS_FOUND=1 \
+        -DBLAS_LIBRARIES="${libdir}/lib${BLAS_LIB}.${dlext}" \
+        -DBLAS_LINKER_FLAGS="${BLAS_LIB}" \
+        -DBLA_VENDOR="${BLAS_NAME}" \
+        -DLAPACK_LIBRARIES="${libdir}/lib${BLAS_LIB}.${dlext}" \
```

Comment:
```suggestion
        -DCBLAS_FOUND=1 \
        -DCBLAS_PROVIDER="libblastrampoline" \
        -DCBLAS_LIBRARIES="${libdir}/lib${BLAS_LIB}.${dlext}" \
        -DCBLAS_INCLUDE_DIRS="/workspace/destdir/include/libblastrampoline/ILP64/${target}" \
        -DLAPACK_LIBRARIES="${libdir}/lib${BLAS_LIB}.${dlext}" \
```

I see that coreblas has a separate find_package step for cblas, so I think we need to add these flags to the build to tell it that the cblas functions are exposed through libblastrampoline as well.

(libblastrampoline should be exporting all the cblas symbols)